Controller/Annotations/AbstractScalarParam.php

```diff
@@ -65,7 +65,9 @@ public function getConstraints()
         }
 
         if (false === $this->allowBlank) {
-            $constraints[] = new NotBlank();
+            $notBlank = new NotBlank();
+            $notBlank->allowNull = $this->nullable;
```

Comment:
We need to do this conditionally as the `allowNull` option didn't exist before Symfony 4.3.

Reply:
I have added the isset($notBlank->allowNull), thanks!